plugins/dell-dock/dell-dock.quirk

```diff
@@ -126,15 +129,32 @@ DellDockHubVersionLowest = 1.31
 DellDockBlobMajorOffset = 0x400a
 DellDockBlobMinorOffset = 0x4009
 
-# USB4 controller
+# USB4 controller (TBT)
 [TBT-00d4b071]
-Name = USB4 controller in Dell dock
+Name = USB4 controller (TBT) in Dell dock
 Summary = USB4 controller
 Vendor = Dell Inc.
 VendorId = TBT:0x00D4
 ParentGuid = USB\VID_413C&PID_B06E&hub&embedded
 FirmwareSizeMin=0x40000
 FirmwareSizeMax=0x80000
-Flags = skips-restart,require-ac,dual-image
+Flags = skips-restart,require-ac,dual-image,updatable
```

Comment:
is this flag change actually needed?  The plugin adds it: https://github.com/fwupd/fwupd/blob/master/plugins/thunderbolt/fu-thunderbolt-device.c#L141